tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -205,6 +205,19 @@ bool isConstantTextureBufferLoad(const Expr *expr) {
   return false;
 }
 
+/// Returns true if the given expr is an DeclRefExpr referencing a kind of
+/// structured or byte buffer and it is non-alias one.
+///
+/// Note: legalization specific code
+bool isAKindOfStructuredOrByteBufferDeclRef(const Expr *expr) {
```

Comment:
we should have a better name for this function as it does more than just checking ``isAKindOfStructuredOrByteBuffer``. e.g. it checks ``isExternalVar``...
how about ``isNonAliasStructuredOrByteBufferVar`` ?

Reply:
Done.